opendevin/llm/bedrock.py

```diff
@@ -5,9 +5,9 @@
 from opendevin.core.config import config
 from opendevin.core.logger import opendevin_logger as logger
 
-AWS_ACCESS_KEY_ID = config.llm.aws_access_key_id
-AWS_SECRET_ACCESS_KEY = config.llm.aws_secret_access_key
-AWS_REGION_NAME = config.llm.aws_region_name
+AWS_ACCESS_KEY_ID = config.get_llm_config().aws_access_key_id
+AWS_SECRET_ACCESS_KEY = config.get_llm_config().aws_secret_access_key
+AWS_REGION_NAME = config.get_llm_config().aws_region_name
```

Comment:
Will it work with bedrock if bedrock vars are set in some llm config, but not default?

Reply:
It only works if it's set as default model. This code was a hack IMO and I don't know how to make it better.

Reply:
We can add some TODO here. Aws-specific stuff is a bit oddly placed on top of our existing vars, and I seem to recall we also want to support this way a few other widely used providers. We may need to think about a better solution how to plug them in.

(also, this code was added when litellm was bugged IIRC, so now users have another option to make it work, although still as default)